CONTRIBUTING.md

```diff
@@ -214,7 +214,7 @@ If you want to have more than one commit in Git master's history after merge (i.
 
 Here is a guide for squashing, if you need help with that: https://redew.github.io/rebaseguide/
 
-If while rebasing, you somehow unintentionally break your pull request, do not close it and make a new one to replace it. Instead, you can ask in the Development chatroom for help on trying to fix it; it can almost always be fixed.
+If while rebasing you somehow unintentionally break your pull request, do not close it and make a new one to replace it. Instead, you can ask in the Development chatroom for help on trying to fix it; it can almost always be fixed.
```

Comment:
```suggestion
If, while rebasing, you somehow unintentionally break your pull request, do not close it and make a new one to replace it. Instead, you can ask in the Development chatroom for help on trying to fix it; it can almost always be fixed.
```

The one comma might be weird, but this is another one of those situations where commas are optional and it adds more clarity with them in than out.